arch/arm64/boot/dts/freescale/beluga-rpmsg.dtsi

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+/*
+ * Copyright 2020 NXP
+ * Copyright (C) 2022 Leica Geosystems AG
+ */
+
+/ {
+	reserved-memory {
+		#address-cells = <2>;
+		#size-cells = <2>;
+		ranges;
+
+		m7_reserved: m7@0x80000000 {
+			no-map;
+			reg = <0 0x80000000 0 0x1000000>;
```

Comment:
Nitpick: Please exchange lines in order, would be consistent with all other nodes.